### ISSUE TITLE
Add support new for ElasticLoadBalancingV2::Listener Action properties

### ISSUE THREAD

**Issue by dgouldin** [cloudtools/troposphere#1240]
CloudFormation's ALB Listener Action documentation now includes several additional properties not yet supported by troposphere:

```
{
  "AuthenticateCognitoConfig" : AuthenticateCognitoConfig,
  "AuthenticateOidcConfig" : AuthenticateOidcConfig,
  "FixedResponseConfig" : FixedResponseConfig,
  "Order" : Integer,
  "RedirectConfig" : RedirectConfig,
  "TargetGroupArn" : String,
  "Type" : String
}
```

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-listener-defaultactions.html

Please add support for these properties. Thanks!

**Comment by jk2l** [comment]
+1

**Comment by markpeek** [comment]
Fixed via #1140
